Configuration/TCA/Overrides/pages.php

```diff
@@ -38,17 +32,7 @@
         'media' => [
             'description' => 'LLL:EXT:newspage/Resources/Private/Language/locallang_be.xlf:media.description',
             'config' => [
-                'overrideChildTca' => [
-                    'columns' => [
-                        'uid_local' => [
-                            'config' => [
-                                'appearance' => [
-                                    'elementBrowserAllowed' => $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext'],
-                                ],
-                            ],
-                        ],
-                    ],
-                ],
+                'allowed' => $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext'],
```

Comment:
for v12 you can use `common-image-types` see https://docs.typo3.org/m/typo3/reference-tca/12.4/en-us/ColumnsConfig/Type/File/Index.html#confval-file-allowed

Reply:
I've changed this line to use `common-image-types`